Libraries/Vibration/Vibration.js

```diff
@@ -27,11 +27,31 @@ var Platform = require('Platform');
  */
 
 var Vibration = {
-  vibrate: function(duration: number = 400) {
+  vibrate: function(pattern: number | Array<number> = 400, repeat: number = -1) {
```

Comment:
@dmmiller I think a number >= 0 is a reasonable API. 0 = don't repeat, n = repeat n times, of course we've to normalize the value to what Android understands.

In fact, the web vibration API doesn't have repeat option at all. I guess because it can be done using a loop and timeout, though not very convenient.